test/mix/tasks/lvn.swiftui.gen.livemarkdown_test.exs

```diff
@@ -60,8 +60,10 @@ defmodule Mix.Tasks.Lvn.Swiftui.LivemarkdownTest do
     )
     ```
     """
-    assert Livemarkdown.make_ex_doc_friendly(content, "filename.livemd") =~
-             "[![Run in Livebook](https://livebook.dev/badge/v1/blue.svg)](https://livebook.dev/run?url=https%3A%2F%2Fraw.githubusercontent.com%2Fliveview-native%liveview-client-swiftui%2Fmain%2Flivebooks%filename.livemd)"
+    result = Livemarkdown.make_ex_doc_friendly(content, "filename.livemd")
+    refute result =~ "Mix.install"
+    assert result =~
+             "[![Run in Livebook](https://livebook.dev/badge/v1/black.svg)](https://livebook.dev/run?url=https%3A%2F%2Fgithub.com%2Fliveview-native%2Fliveview-client-swiftui%2Fblob%2Fmain%2Flivebooks%2Ffilename.livemd)"
```

Comment:
what is the value of this type of test?

Reply:
It ensures the conversion from livemd to md works as expected and documents the code's behavior/purpose. I used it to drive the implementation of the code TDD style.

However, I the value add is relatively minimal and it's one more thing to upkeep. I'm fine to remove it if you prefer.

Reply:
Got it, that's ok then.